guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -21,22 +21,22 @@ ifdef::foreman-el,katello,satellite[]
 |====
 |Directory |Installation Size |Runtime Size
 
-|/var/log/ |10 MB |10 GB
+|/var/log |10 MB |10 GB
 
-|/var/lib/pgsql |100 MB |20 GB
+|/var/lib/pgsql |100 MB |20 GB
 
-|/usr | 3 GB | Not Applicable
+|/usr | 5 GB | Not Applicable
 
 |/opt/puppetlabs | 500 MB | Not Applicable
 
 ifdef::katello,satellite,orcharhino[]
-|/var/lib/pulp/ |1 MB |300 GB
+|/var/lib/pulp |1 MB |300 GB
 
-|/var/lib/qpidd/ |25 MB | Not Applicable
+|/var/lib/qpidd |25 MB | xref:storage-guidelines_{context}[Refer Storage Guidelines]
 endif::[]
 |====
 
-For external database servers: `/var/lib/pgsql` with installation size of 100 MB and runtime size of 20 GB.
+For detailed information on partitioning and size, refer to the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/partitioning-reference_system-design-guide[{RHEL} 8 partitioning guide].
```

Comment:
This reverts the changes made in https://github.com/theforeman/foreman-documentation/pull/1684, which seems wrong.